dora/shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -229,7 +238,8 @@ public static String description() {
         + "    metrics          metrics information\n"
         + "    summary          cluster summary\n"
         + "    ufs              under storage system information\n"
-        + "    jobservice       job service metrics information\n";
+        + "    jobservice       job service metrics information\n"
+        + "    nodestatus       node status [worker as of now]\n";
```

Comment:
feel free to leave it next pr, but we shall just call this cmd either `status` or `nodes` (`workers`).

Reply:
[ADDRESS IN NEXT PR]

Reply:
It's not part of ReportCommand anymore, I have created another fsadmin cmd, named "nodes", we can do showstatus or other admin functionalities such as remove nodes over there.
introduced in #17890
e.g.
$bin/alluxio fsadmin nodes status